packages/swarm-constants/properties/size/spacing.json

```diff
@@ -20,6 +20,10 @@
             "quadruple": {
                 "value": "64px",
                 "comment": "quadruple space size"
+            },
+            "largeDefault": {
```

Comment:
There should be quarter, half, double, & quadruple spaces defined here as well as the default.